pom.xml

```diff
@@ -257,6 +257,11 @@
                 <artifactId>wildfly-naming-client</artifactId>
                 <version>${version.org.wildfly.naming.client}</version>
             </dependency>
+            <dependency>
```

Comment:
I am not sure what to do here. We do have two versions of naming client, but the wildfly-http-naming-client has only one version, because it does not depend on java EE. Isn't this weird?

Reply:
The reason is previously the wildfly-http-client became interoperable if and only if the system property was set.
There was no need to have two versions of wildfly-http-client. This has changed when you introduced 
EENamespaceInteroperability.VARIABLE_CONSTANT magic constant @fl4via . We need to provide two versions now.

Reply:
This is a temporary problem @fl4via . It will be fixed once we will provide only jakarta versions of our libraries.